rest/src/main/kotlin/builder/component/ActionRowBuilder.kt

```diff
@@ -44,12 +44,73 @@ public class ActionRowBuilder : MessageComponentBuilder {
      * Creates and adds a select menu with the [customId] and configured by the [builder].
      * An ActionRow with a select menu cannot have any other select menus or buttons.
      */
-    public inline fun selectMenu(customId: String, builder: SelectMenuBuilder.() -> Unit) {
+    @Deprecated("Renamed by discord to StringSelect", ReplaceWith("stringSelect"))
```

Comment:
```suggestion
    @Deprecated("Renamed by discord to StringSelect", ReplaceWith("stringSelect(customId, builder)"))
```
will fill parameters when using automatic replacement in ide